scenarios/getting_started/go/getting_started.go

```diff
@@ -0,0 +1,110 @@
+package main
+
+import (
+	"context"
+	"crypto/tls"
+	"fmt"
+	"log"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/eclipse/paho.golang/paho"
+	"github.com/joho/godotenv"
+)
+
+type MqttConnectionSettings struct {
+	Hostname  string
+	TcpPort   int
+	UseTls    bool
+	CaFile    string
+	KeepAlive uint16
+	ClientId  string
+	Username  string
+	KeyFile   string
+}
+
+func main() {
+	// Load connection settings
+	godotenv.Load("../.env")
+	cs := MqttConnectionSettings{
+		Hostname:  os.Getenv("MQTT_HOST_NAME"),
+		TcpPort:   8883,
+		Username:  os.Getenv("MQTT_USERNAME"),
+		ClientId:  os.Getenv("MQTT_CLIENT_ID"),
+		CaFile:    os.Getenv("MQTT_CERT_FILE"),
+		KeepAlive: 30,
+		KeyFile:   os.Getenv("MQTT_KEY_FILE"),
+	}
+
+	// Load certificates
+	fmt.Println("Loading certificates")
+	cert, err := tls.LoadX509KeyPair(fmt.Sprintf("../%s", cs.CaFile), fmt.Sprintf("../%s", cs.KeyFile))
```

Comment:
mmhhh LoadX509KeyPair seems needed to load the private key, not needed here. Is there any LoadPem? or similar